### PR TITLE
Bubuku must live even after catching exceptions

### PR DESCRIPTION
**Pull request by antban**

### CODE REVIEW

File: bubuku/config.py [hunk 1]
```diff
@@ -72,5 +72,7 @@ def _make_clean_line(l: str) -> str:
     result = l.strip()
     if result.startswith('#') or not result:
         return result
+    if '=' not in result:
```
Comment (rcillo) [inline bubuku/config.py]: How is this related to the fix?

**Comment by rcillo** [comment]
:shipit:

File: bubuku/config.py (hunk 1 above)
Comment (antban) [inline bubuku/config.py]: No it's not related, but it fixes some issues
